ciscoasa/objects_networkservicegroups.go

```diff
@@ -169,10 +179,6 @@ func (s *objectsService) UpdateNetworkServiceGroup(name, description string, mem
 	}
 
 	for _, member := range members {
-		if !strings.Contains(member, "/") {
-			member = fmt.Sprintf("/%s", member)
-		}
-
```

Comment:
Ah yes, I actually meant for that to be in a separate pull request. But the Asa (and library) allow service groups to be nested. A group name will not contain a slash, so nesting groups didn't work anymore because of that.